scripts/toggle_logging.sh

```diff
@@ -5,21 +5,18 @@ CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source "$CURRENT_DIR/variables.sh"
 source "$CURRENT_DIR/shared.sh"
 
-get_filename() {
-	local logging_path="$(get_tmux_option "$logging_path_option" "$default_logging_path")"
-	local logging_filename="$(get_tmux_option "$logging_filename_option" "$default_logging_filename")"
-	echo "${logging_path}/${logging_filename}"
-}
 
 start_pipe_pane() {
-	local filename="$(get_filename)"
-	"$CURRENT_DIR/start_logging.sh" "$filename"
-	display_message "Started logging to $filename"
+	local file=$(tmux display-message -p "$logging_full_filename")
+	local path=$(tmux display-message -p "$logging_path")
```

Comment:
Seing these 2 lines with `tmux display-message -p` again tripped me, despite getting your clarification in #20.
Chances are anyone else reading the source will be puzzled as well. How about extracting `tmux display-message -p` to a helper function? The purpose would be just to clarify intention. Example names: `expand_value`, `expand_tmux_formats`.. or something more fitting

Reply:
I think about this part... Maybe you are right and extracting this code to external function will be a good decision